package.json

```diff
@@ -36,6 +36,8 @@
     "generate:types:dev": "openapi-typescript -c .prettierrc https://api.dev.getoctane.io/docs/openapi.json --output src/apiTypes.ts"
   },
   "peerDependencies": {
+    "@stripe/react-stripe-js": "*",
+    "@stripe/stripe-js": "*",
```

Comment:
We probably want at least the same versions that we used to support, rather than any version at all. If Stripe releases a v2 of either of these, we'd probably need to upgrade this library anyways.